docs/ops/experimental/ExperimentalDetectronROIFeatureExtractor_6.md

```diff
@@ -0,0 +1,125 @@
+## ExperimentalDetectronROIFeatureExtractor <a name="ExperimentalDetectronDetectionOutput"></a> {#openvino_docs_ops_experimental_ExperimentalDetectronROIFeatureExtractor_6}
```

Comment:
Maybe you should write the PR title as 'Add specification for `ExperimentalDetectronROIFeatureExtractor`, because this PR doesn't contain other Detectron-like operations specifications?

Reply:
If previous my comment is not correct, please, add other specifications.

Reply:
All operations specifications will be added in this review. I will add rest operations without detailed descriptions today or tomorrow.